github/release_notes/model.py

```diff
@@ -28,19 +29,60 @@
     "is_current_repo",
     "from_same_github_instance",
     "cn_source_repo"
-])
+])):
+    def __eq__(self, other):
```

Comment:
why is this required? namedtuple instances should already have `__eq__` implementations with this sematics (equal iff all attrs are equal).

also, consider switching to data classes (new feature in python3.7):

https://docs.python.org/3/library/dataclasses.html